styles/alert.css

```diff
@@ -22,3 +22,102 @@
   --bs-alert-color: rgb(var(--stanford-illuminating-dark-rgb));
   --bs-alert-border-color: rgb(var(--stanford-illuminating-dark-rgb));
 }
+
+/* Banner styles */
+.banner {
+  --banner-bg: none;
+  --banner-padding-x: 16px;
+  --banner-padding-y: 16px;
+  --banner-margin-bottom: 1rem;
+  --banner-color: inherit;
```

Comment:
don't think this is necessary unless you want to apply a banner that isn't any of these, and even if it just won't do a color.